extensions/sql-migration/src/constants/strings.ts

```diff
@@ -380,6 +380,8 @@ export const DETAILS_COPIED = localize('sql.migration.details.copied', "Details
 export const CANCEL_MIGRATION_CONFIRMATION = localize('sql.cancel.migration.confirmation', "Are you sure you want to cancel this migration?");
 export const YES = localize('sql.migration.yes', "Yes");
 export const NO = localize('sql.migration.no', "No");
+export const EMPTY_TABLE_TEXT = localize('sql.migration.empty.table.text', "No backup files to show");
+export const EMPTY_TABLE_SUBTEXT = localize('sql.migration.emty.table.subtext', "If you expected to see results, please check the connection to the sql server instance");
```

Comment:
"SQL Server" should be capitalized.  Does this string also refer to other flavors of SQL (Azure, Edge, etc.)?  If so, it should be run by a PM to figure out the correct wording.

Reply:
For now, let's update the string to: "If results were expected, reconfirm the connection to the SQL Server Instance."
We want to avoid using the word "please" in our strings.
I will run this by our content writer next week. 

NIT: emty -> empty